src/sonic_ax_impl/mibs/ietf/rfc1213.py

```diff
@@ -82,6 +106,19 @@ def update_data(self):
             subid = (if_index,) + iptuple
             self.arp_dest_map[subid] = machex
             self.arp_dest_list.append(subid)
+
+    def update_data(self):
+        self.arp_dest_map = {}
+        self.arp_dest_list = []
+        """
+        update arp table of host.
+        In case of multi-asic platform, get host arp table
+        from kernel and namespace arp table from NEIGH_TABLE in APP_DB
+        in each namespace.
+        """
+        self._update_host_data()
+        if len(self.db_conn) > 1:
```

Comment:
>if len(self.db_conn) > 1: [](start = 8, length = 25)

You may remove this check. If only one, `_update_namespace_data` will return quickly.

Reply:
done.